src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpMetricExporterExtensions.cs

```diff
@@ -60,6 +61,8 @@ public static MeterProviderBuilder AddOtlpExporter(this MeterProviderBuilder bui
 
             OtlpExporterOptions.RegisterOtlpExporterOptionsFactory(services);
 
+            services.RegisterOptionsFactory(configuration => new ExperimentalOptions(configuration));
```

Comment:
nit: It seems like all signals are now using `ExperimentalOptions`. You could refactor the [helper called above](https://github.com/open-telemetry/opentelemetry-dotnet/blob/4cccb09239aff1250967c543f7483aaf88e538a3/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs#L225-L228) to do this registration to avoid having to add an extra call everywhere.

Something like this...

```diff
-   internal static void RegisterOtlpExporterOptionsFactory(IServiceCollection services)
+   internal static void RegisterOtlpExporterSharedServices(IServiceCollection services)
    {
        services.RegisterOptionsFactory(CreateOtlpExporterOptions);
+       services.RegisterOptionsFactory(configuration => new ExperimentalOptions(configuration));
    }
```

Reply:
done